test/test_jit_expression.py

```diff
@@ -208,3 +209,47 @@ def exact_expr(x):
     exact = exact_expr(points.T)
 
     assert np.allclose(exact, output)
+
+
+def test_grad_constant(compile_args):
+    """
+    Test if numbering of constants are correct after UFL eliminates the constant inside the gradient
+    """
```

Comment:
```suggestion
    """Test if numbering of constants are correct after UFL eliminates the constant inside the gradient."""
```
(pydocstyle will complain about this after #667 is merged)